circuit-foundation/src/commonMain/kotlin/com/slack/circuit/foundation/Circuit.kt

```diff
@@ -184,7 +184,7 @@ public class Circuit private constructor(builder: Builder) {
 private val UnavailableContent: @Composable (screen: Screen, modifier: Modifier) -> Unit =
   { screen, modifier ->
     BasicText(
-      "Route not available: ${screen::class.qualifiedName}",
+      "Route not available: $screen",
```

Comment:
This is the only place I noted that actually uses the `data` class `toString` method (I may have missed something). Do we actually want to go down the path of recommending screens be data classes? I'm worried we'd make it easier to unintentionally display and/or leak screen data that should remaining hidden. Data classes bring a lot of capability that isn't really needed at the moment (ie. copy, equals, hashcode).

Were there any other options under consideration?

Reply:
We already use data classes for screens, the big diff now it's that we have data objects now and didn't before for easy object toStrings

Reply:
Nothing more appropriate comes to mind so there's no point in holding this back